documents/src/pages/elements/datetime-picker.md

```diff
@@ -343,6 +347,303 @@ section {
 </ef-datetime-picker>
 ```
 
+### Custom cells
+Datetime Picker allows you to customise cells on the calendar using slots. Each date cell provides slot with name in `yyyy-MM-dd` format. In case of year or month cell, the format is `yyyy` or `yyyy-mm`.
```

Comment:
```suggestion
Datetime Picker allows you to customise cells on the calendar using slots. Each date cell provides slot with name in `yyyy-MM-dd` format. In case of year or month cell, the format is `yyyy` or `yyyy-MM`.
```